lib/ansible/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -898,7 +906,7 @@ def main():
         proxmox = ProxmoxAPI(api_host, user=api_user, password=api_password, verify_ssl=validate_certs)
         global VZ_TYPE
         global PVE_MAJOR_VERSION
-        PVE_MAJOR_VERSION = 3 if float(proxmox.version.get()['version']) < 4.0 else 4
+        PVE_MAJOR_VERSION = 3 if proxmox_version(proxmox) < 4.0 else 4
```

Comment:
I think #59356 handles this better. Could we combine them?

Reply:
@apple4ever I can imagine, `int` might not be sufficient, but still. `int` is way better than float.
I would use `int` over `tuple`, at least.

Reply:
@limakzi I also liked that it set it to the right version instead of just defaulting to 4. So 5 would be 5 and 6 would be 6.

Reply:
@mrdrogdrog could you make that change? 

If you can do that, I will find who needs to sign off so we can get this merged and released.

Reply:
I would change this to:

`PVE_MAJOR_VERSION = int(proxmox.version.get()['version'].split('.', 1)[0])`

Reply:
How about using `LooseVersion` from `distutils.version`? It usually handles version numbers pretty well. (Grep for `LooseVersion` in the modules directory to see plenty of examples.)

Reply:
I really like the suggestion with `LooseVersion`. I'm going to adjust my code when I have free time. Thx @felixfontein !

Reply:
@mrdrogdrog great! Once you made the adjustments and someone else also tested it (just to be sure), I'll merge it and create a backport PR. If this gets done by the end of next week, it should make it into the 2.9.2 release.